src/libstd/keyword_docs.rs

```diff
@@ -991,9 +991,49 @@ mod pub_keyword {}
 //
 /// Bind by reference during pattern matching.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// `ref` annotates pattern bindings to make them borrow rather than move.
+/// It is **not** a part of the pattern as far as matching is concerned.
```

Comment:
I think I understand what you're trying to say - the same values are matched with or without `ref` - but it's not very clear from the wording. Maybe you can say "this does not affect _whether_ a value is matched, only _how_ it is matched"?

Reply:
Done 👍